### PR TITLE
Add timezone support

### PR DESCRIPTION
**Pull request by mpuckett159**
Updating message_timestamp to support specifying both timezone aware and naive datetimes, allowing us to include timezone information in tweets and allowing users to specify their desired timezone

Updating .env file to include the new environment configuration file.

Updating requirements to include pytz which is how I'm supporting timezone information.

Updating env.py import to include the new environment variable as well as include a default configuration.

Fixing datetime string format for tweets, reverting prior erroneous change.

### CODE REVIEW

**Comment by AetherUnbound** [comment]
This looks like a great set of changes! We discussed this a bit offline, but I'll enumerate it here: I think it might be best to remove the timestamp from the Twitter message entirely for a few reasons.

1. Twitter has the timestamp of when the tweet was made as part of the tweet metadata, so the manual timestamp is redundant.
2. The timestamp is extracted from the Signal message payload as the time the Signal message was sent. This may not correspond with the time the broadcast was issued, and adding that to the tweet may be misleading (as the broadcast could be heard minutes after it was actually captured).

Would it be possible to modify this PR so that the timestamp is no longer added to the final tweet? We will also need to check if the `_f_not_recent` filter still needs to be converted. I believe that all timestamps are extracted as UTC, and so if the system clock is on PDT or something similar, _all_ messages will be seen as "not recent" since there's a 7 hour time difference between those timezones.

**Comment by mpuckett159** [comment]
Removing the timestamp is pretty trivial at least, just gotta remove the line from the tweet formatting string and the input because it's unnecessary now.

The `_f_not_recent` function should be unaffected. The way changed the `message_timestamp` function was to make use of the `convert` input to decide whether to include timezone data or not, and the output that is used by `_f_not_recent` was generated without the timezone information, so after removing all the timezone updates it'll just be back to the way it was before.

File: .env.example [hunk 1]
```diff
@@ -4,4 +4,5 @@ LISTEN_GROUP=signal groupId for the group to listen to
 TWITTER_API_KEY=Twitter API key
 TWITTER_API_SECRET=Twitter API secret
 TWITTER_ACCESS_TOKEN=Twitter access token
-TWITTER_TOKEN_SECRET=Twitter access token secret
\ No newline at end of file
+TWITTER_TOKEN_SECRET=Twitter access token secret
+TIMEZONE=tz_database format timezone (see https://en.wikipedia.org/wiki/List_of_tz_database_time_zones for complete list, default is UTC)
```
Comment (AetherUnbound) [inline .env.example:8]: I think that this env var may not be needed any more since no conversions are being done

File: requirements.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 Click==7.1.2
 tweepy==3.9.0
-ujson==3.1.0
\ No newline at end of file
+ujson==3.1.0
+pytz==2020.1
```
Comment (AetherUnbound) [inline requirements.txt:4]: Is this requirement still needed? I don't see it used in any of the affected files

File: signal_scanner_bot/env.py [hunk 3]
```diff
@@ -42,7 +42,7 @@ def log_vars() -> None:
 TWITTER_ACCESS_TOKEN = _env("TWITTER_ACCESS_TOKEN")
 TWITTER_TOKEN_SECRET = _env("TWITTER_TOKEN_SECRET")
 TRUSTED_TWEETERS = set(str(_env("TRUSTED_TWEETERS", default="")).split(","))
-TZ_UTC = _env("TZ_UTC", fail=False)
+TIMEZONE = _env("TIMEZONE", fail=False, default="UTC")
```
Comment (AetherUnbound) [inline signal_scanner_bot/env.py:45]: Same comment RE converting

**Review by AetherUnbound — COMMENTED** [review]
Thanks for adjusting this PR! Just a couple more thoughts based on how we ended up going with these changes.

File: .env.example (hunk 1 above)
Comment (mpuckett159) [reply .env.example]: Whoops, correct, cleaning these up.

**Review by AetherUnbound — APPROVED** [review]
Looks great! Thanks again!

**Pull request merged**